crates/astria-core/src/execution/v1alpha2/mod.rs

```diff
@@ -104,7 +116,9 @@ impl Protobuf for GenesisInfo {
                  under the hood",
             );
         Self::Raw {
-            rollup_id: Bytes::copy_from_slice(rollup_id.as_ref()),
+            rollup_id: Some(RawRollupId {
+                inner: Bytes::copy_from_slice(&rollup_id.inner),
+            }),
```

Comment:
```suggestion
            rollup_id: Some(rollup_id.to_raw()),
```
you can rely on the existing `RollupId::to_raw()` implementation, instead of recreating here and then do not need to import the RawRollupId type